bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py

```diff
@@ -12,9 +12,10 @@ class PyMultiscaleRun(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/-/ide/project/molsys/multiscale_run"
     git = "ssh://git@bbpgitlab.epfl.ch/molsys/multiscale_run.git"
 
-    maintainers("tristan0x")
+    maintainers("tristan0x, katta")
```

Comment:
```suggestion
    maintainers("tristan0x", "cattabiani")
```

Should be two maintainers, not one maintainer called "tristan0x, katta" :-)
It seems convention to use your github name, but I'll leave it up to you whether you want to actually accept that part of the suggestion.